motr/idx.c

```diff
@@ -206,6 +208,7 @@ static int idx_op_init(struct m0_idx *idx, int opcode,
 	oi->oi_vals = vals;
 	oi->oi_rcs  = rcs;
 	oi->oi_flags = flags;
+	oi->oi_min_success = 1;
```

Comment:
I am thinking whether we should set this to M0_DIX_MIN_REPLICA_QUORUM or to 1.
From the code in st_idx_op(), I can see M0_DIX_MIN_REPLICA_QUORUM is the default value we should have.

What do you think?

Reply:
I was hesitating to make `M0_DIX_MIN_REPLICA_QUORUM` the default because some of the unit tests are sensitive to this kind of behavior, but now that I think about it they should be setting min_success separately from the main client interface. I'm running unit tests now to confirm that `oi->oi_min_success = M0_DIX_MIN_REPLICA_QUORUM` works.

I think `M0_DIX_MIN_REPLICA_QUORUM` would be a better choice since it gives basic read-after-write consistency by default. This means the RGW folks wouldn't need to make any immediate changes (can still add options to configure manually later).

Reply:
Good.